Doc/whatsnew/3.9.rst

```diff
@@ -666,7 +666,11 @@ Removed
   module have been removed.  They were deprecated in Python 3.2.
   Use ``iter(x)`` or ``list(x)`` instead of ``x.getchildren()`` and
   ``x.iter()`` or ``list(x.iter())`` instead of ``x.getiterator()``.
-  The ``xml.etree.cElementTree`` module has been removed.
+  The ``xml.etree.cElementTree`` module has been removed,
+  use the ``xml.etree.ElementTree`` module instead.
```

Comment:
We can add a reference to an existing module (but only one per paragraph, so the second mention should not be changed).
```suggestion
  use the :mod:`xml.etree.ElementTree` module instead.
```

Reply:
We can (and we should). Thanks.